media-gfx/gnome-photos/metadata.xml

```diff
@@ -14,7 +14,6 @@
     You can:
     - Automatically find all your pictures
     - View recent local and online photos
-    - Access your Facebook or Flickr pictures
```

Comment:
I'm dropping this change, because this text still appears in `data/org.gnome.Photos.appdata.xml.in`. I believe *oauth* support was removed for facebook and/or flickr.